torch/distributed/distributed_c10d.py

```diff
@@ -1714,6 +1714,8 @@ def broadcast_object_list(object_list, src=0, group=None):
         src (int): Source rank from which to broadcast ``object_list``.
         group: (ProcessGroup, optional): The process group to work on. If None,
             the default process group will be used. Default is ``None``.
+        device (``torch.device``, optional): device to send from or receive
```

Comment:
any reason this doc arg name does not math the API arg name?

Reply:
great catch